spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -275,17 +275,32 @@ def show_total_results(self):
                     # Note that the midsagittal slice is the same for all lesions as it is based on the spinal cord
                     # segmentation
                     midsagittal_slice = str(int(row['midsagittal_spinal_cord_slice']))
-                    # Get dorsal and ventral tissue bridges for the mid-sagittal slice
-                    dorsal_tissue_bridge = row[f'slice_{midsagittal_slice}_dorsal_bridge_width [mm]']
-                    ventral_tissue_bridge = row[f'slice_{midsagittal_slice}_ventral_bridge_width [mm]']
+                    if idx == 0:        # Print only once, not for each lesion
+                        printv(f'\nMinimum tissue bridges across all lesions for the midsagittal slice '
+                               f'(sagittal slice {midsagittal_slice})...', self.verbose, 'normal')
+                    # Check whether the lesion has bridges in the midsagittal slice, if not, set the bridge width to NaN
+                    if f'slice_{midsagittal_slice}_dorsal_bridge_width [mm]' in row and \
+                            f'slice_{midsagittal_slice}_ventral_bridge_width [mm]' in row:
+                        # Get dorsal and ventral tissue bridges for the mid-sagittal slice
+                        dorsal_tissue_bridge = row[f'slice_{midsagittal_slice}_dorsal_bridge_width [mm]']
+                        ventral_tissue_bridge = row[f'slice_{midsagittal_slice}_ventral_bridge_width [mm]']
+                    else:
+                        dorsal_tissue_bridge = np.nan
+                        ventral_tissue_bridge = np.nan
+                    # If there are NaN values, print a warning
+                    if np.isnan(dorsal_tissue_bridge) or np.isnan(ventral_tissue_bridge):
+                        printv(f'WARNING: Lesion #{idx+1} does not exist in the midsagittal slice',
+                               self.verbose, type='warning')
```

Comment:
I think there's no need for this `if` statement, right? 

```suggestion
                        dorsal_tissue_bridge = np.nan
                        ventral_tissue_bridge = np.nan
                        printv(f'WARNING: Lesion #{idx+1} does not exist in the midsagittal slice',
                               self.verbose, type='warning')
```

Reply:
Good question! I tried to clarify why I'm using the extra `if` in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4595/commits/2dafd6c28b2aa7ebc45b5223b90e504129234215

Here is a supporting screenshot of `self.measure_pd`:

<img width="1485" alt="image" src="https://github.com/user-attachments/assets/772516cf-e63d-4d83-a44e-bc02e5a9ecbc">

Reply:
Ahhhhh, thank you for clarifying!! I understand now. :)